microsoft-edge/extensions-chromium/publish/contact-extensions-team.md

```diff
@@ -6,69 +6,76 @@ ms.author: msedgedevrel
+* Discuss developing extensions.
+   * Share best practices with other developers on building, managing or acquiring more users for their browser extension.
+   * Connect with other extension developers about technical questions related to building Microsoft Edge add-ons.
+   * Share ideas about the Edge Add-ons website (the Edge extensions store).
+* Request a feature for extensions.
+   * Share inputs or suggestions to the Microsoft Edge add-ons engineering team.
+   * Discuss how to improve extension publishing, management and listing processes or workflows.
+   * Provide early feedback to the Microsoft Edge add-ons engineering team about new features for managing and publishing extensions.
 
-You can contact the Microsoft Edge extensions support team to request help, submit feedback, or report a copyright or trademark infringement.
+
+<!-- ====================================================================== -->
+## Issues page in the MicrosoftEdge-Extensions repo
+
+To publicly report or discuss an issue about extensions, use the [Issues](https://github.com/microsoft/MicrosoftEdge-Extensions/issues) page in the **MicrosoftEdge-Extensions** repo.
+
+Use this **Issues** page for: 
+* Aspects of Microsoft Partner Center that affect all extension developers.
+* Aspects of the Edge Add-ons website that affect all extension developers or all Microsoft Edge extension users.
 
 
 <!-- ====================================================================== -->
-## Inquire about your extension's review status or certification status
+## Send email to extensions developer support
 
-After submitting your extension via Partner Center, if you have questions regarding your extension's review status or certification status, use the [MicrosoftEdge-Extensions](https://github.com/microsoft/MicrosoftEdge-Extensions/issues/new/choose) repo to enter a new Issue.
+To discuss issues about a specific developer account or extension, you can privately contact extensions developer support at [ext_dev_support@microsoft.com](mailto:ext_dev_support@microsoft.com).
 
 
 <!-- ====================================================================== -->
-## Learn about correct use of APIs
+## Inquire about an extension's review status or certification status
 
-If your extension isn't working with Microsoft Edge as you expect, or you have a question about how to develop an extension, see [Supported APIs for Microsoft Edge extensions](../developer-guide/api-support.md) and [Port Chromium extensions to Microsoft Edge](../developer-guide/port-chrome-extension.md).
+After submitting your extension via Partner Center, if you have questions regarding your extension's review status or certification status, use the [MicrosoftEdge-Extensions](https://github.com/microsoft/MicrosoftEdge-Extensions/issues/new/choose) repo to enter a new Issue.
```

Comment:
inquiring about a specific submission should be done privately not via public Issues area of repo.
@bhuvanapriyap

Reply:
fixed